#@module_name hsr_carry
string  destination
---
bool success